src/api/java/com/minecolonies/api/util/Tuple.java

```diff
@@ -1,15 +1,31 @@
 package com.minecolonies.api.util;
 
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Our own tuple implementation with hashcode and equals.
  * @param <A> The first obj.
  * @param <B> The second obj.
  */
-public class Tuple<A, B> extends net.minecraft.util.Tuple<A, B>
+public class Tuple<A, B>
 {
-    public Tuple(final A aIn, final B bIn)
+    private A a;
+    private B b;
+
+    public Tuple(@Nullable final A aIn, @Nullable final B bIn)
     {
-        super(aIn, bIn);
+        this.a = aIn;
+        this.b = bIn;
+    }
+
+    @Nullable
+    public A getA() {
+        return this.a;
+    }
+
+    @Nullable
+    public B getB() {
+        return this.b;
     }
 
     @Override
```

Comment:
if (o instanceof net.minecraft.util.Tuple)
The equals function below is still set for minecraft tuples

Reply:
Good find